analyzer/tests/functional/analyze_and_parse/test_analyze_and_parse.py

```diff
@@ -168,7 +168,12 @@ def check_one_file(self, path, mode):
         post_processed_output = []
         skip_prefixes = ["[] - Analysis length:",
                          "[] - Previous analysis results",
-                         "[] - Skipping input file"]
+                         "[] - Skipping input file",
+                         # Enabled checkers are listed in the beginning of
+                         # analysis.
+                         "[] - Enabled checkers:",
+                         "clang-tidy:",
+                         "clangsa"]
```

Comment:
Do not we need a `:` at the end of this string same as for `clang-tidy`:

```suggestion
                         "clangsa:"]
```